src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -79,7 +79,7 @@
     @ManyToOne(fetch = FetchType.LAZY)
     @JsonView(QuizView.Before.class)
     // TODO: rename ignored property to "results" after change in client
```

Comment:
Should the comment be removed since you renamed the ignored property?

Reply:
Yes, we removed it